apps/synchronize/src/github/eventHelpers.js

```diff
@@ -0,0 +1,78 @@
+import { Account, Plan, Organization, User } from "@argos-ci/database/models";
+
+export async function getAccount(payload) {
+  const { type, id: githubId } = payload.marketplace_purchase.account;
+
+  if (!type) throw new Error(`can't find account without account type`);
+  if (!githubId) throw new Error(`can't find account without githubId`);
+  if (type.toLowerCase() === "user") {
+    const account = await Account.query()
+      .joinRelated("user")
+      .findOne("user.githubId", githubId);
+    return account ?? null;
+  }
+  if (type.toLowerCase() === "organization") {
+    const account = await Account.query()
+      .joinRelated("organization")
+      .findOne("organization.githubId", githubId);
+    return account ?? null;
+  }
+  if (!type) throw new Error(`can't find account of type : ${type}`);
```

Comment:
I think this test makes no sense. You can just throw the error here.

Reply:
yes